opennlp-tools/src/main/java/opennlp/tools/cmdline/namefind/TokenNameFinderTrainerTool.java

```diff
@@ -115,7 +114,7 @@ public void run(String format, String[] args) {
 
     mlParams = CmdLineUtil.loadTrainingParameters(params.getParams(), true);
     if (mlParams == null) {
-      mlParams = new TrainingParameters();
```

Comment:
Should the public constructor for `TrainingParameters` be made private?

Reply:
We probably set default somewhere in case they are missing. This change here would set the defaults to the default training parameters.